translate/storage/test_php.py

```diff
@@ -762,6 +762,17 @@ def test_quotes(self):
         phpfile = self.phpparse(phpsource)
         assert phpfile.__bytes__() == phpsource
 
+    def test_concatenation(self):
+        """Check that a simple definition after define is parsed correctly."""
+        phpsource = """
+$lang['mediaselect'] = "Really \\\"something\\\"" . $variable . "\\\"something else\\\"";"""
+        phpfile = self.phpparse(phpsource)
+        print(len(phpfile.units))
```

Comment:
Unnecessary `print` call.

Reply:
It's used all over the file this way, so it's there after copying existing code. Will remove that in separate commit.

Reply:
Thanks.